engine/src/main/java/org/terasology/config/flexible/FlexibleConfigManager.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.config.flexible;
+
+import org.terasology.engine.SimpleUri;
+
+/**
+ * Stores multiple {@link FlexibleConfig} instances that can be retrieved using their id.
+ * Also responsible for overseeing their serialization - to and from - disk.
```

Comment:
In addition to my previos comment about save/load, I'd expect a method like `save(FlexibleConfig)` and include the reader/writer logic here.

Reply:
Hmmmm... I'm not sure I agree with this and the related comment above. 

Can you do a comparison between the two different ways of doing things and show why the changes you suggest would be better?